tests/rkt_pid_file_test.go

```diff
@@ -121,14 +121,14 @@ func TestPidFileAbortedStart(t *testing.T) {
 	if err := runChild.SendLine("\035\035\035"); err != nil {
 		t.Fatalf("Failed to terminate the pod: %v", err)
 	}
-	if err := runChild.Wait(); err.Error() != "exit status 1" {
-		t.Fatalf("rkt didn't terminate as expected: %v", err)
+	if err := runChild.Wait(); err == nil || err.Error() != "exit status 1" {
+		t.Fatalf("rkt didn't terminate as expected (expects: \"exit status 1\") got: %v", err)
```

Comment:
So when we exit without error we get `rkt didn't terminate as expected (expects: "exit status 1") got: <nil>`. Not really informative, although definitely better than panicking :)

Maybe we can separate both cases and print something better for the nil case?